### PR TITLE
Document how to do unit placement in modules

### PR DESCRIPTION
**Pull request by fridex**
## This introduces a breaking change

- [x] No

### CODE REVIEW

File: docs/source/unit.rst [hunk 1]
```diff
@@ -108,8 +108,45 @@ Pipeline units of type :class:`Sieve <thoth.adviser.sieve.Sieve>` and
...
 
 All pipeline units should be atomic pieces and `they should do one thing and do
 it well <https://en.wikipedia.org/wiki/Unix_philosophy>`_. They were designed
 to be small pieces forming complex resolution system.
+
+Unit placement in a pipeline
+============================
+
+The pipeline configuration (which pipeline units in what configuration) is
+determined dynamically on each adviser start. This enables to construct the
```
Comment (KPostOffice) [inline docs/source/unit.rst:125]: This enables construction of the pipeline depending on an input vector (...)

File: docs/source/unit.rst [hunk 2]
```diff
@@ -108,8 +108,45 @@ Pipeline units of type :class:`Sieve <thoth.adviser.sieve.Sieve>` and
...
+pipeline depending on the input vector (e.g. packages used, Python indexes
+configured, library usage, recommendation type and such). Each pipeline unit is
+asked to be registered to the pipeline configuration until there is any change
```
Comment (KPostOffice) [inline docs/source/unit.rst:128]: Each pipeline unit requests to be registered to the pipeline configuration until the pipeline configuration is stable, meaning that all pipeline units have been registered in the pipeline configuration?

or

Each pipeline unit requests to be registered to the pipeline configuration until the pipeline configuration has been changed, indicating that the unit has been registered?

File: docs/source/unit.rst (hunk 1 above)
Comment (fridex) [reply docs/source/unit.rst]: Adjusted.

Comment (fridex) [reply docs/source/unit.rst]: Adjusted :+1:

File: docs/source/unit.rst (hunk 2 above)
Comment (fridex) [reply docs/source/unit.rst]: I like the latter one as not all units need to be present in the pipeline configuration. Changed, thanks :+1:

**Pull request merged**